GVFS/GVFS.Common/ProductUpgrader.Shared.cs

```diff
@@ -19,12 +18,25 @@ public static bool IsLocalUpgradeAvailable()
             string downloadDirectory = GetAssetDownloadsPath();
             if (Directory.Exists(downloadDirectory))
             {
-                const string PotentialInstallerName = "*VFS*.*";
-                string[] installers = Directory.GetFiles(
-                    downloadDirectory,
-                    PotentialInstallerName, 
-                    SearchOption.TopDirectoryOnly);
-                return installers.Length > 0;
+                // This method is used Only by Git hooks. Git hooks does not have access
+                // to GVFSPlatform to read platform specific file extensions. That is the

```

Comment:
This comment is not true, the hooks access parts of GVFSPlatform already today.

Reply:
Ya... removed the comment.